model/qti/container/Container.php

```diff
@@ -208,7 +209,7 @@ public function checkIntegrity($body, &$missingElements = null)
      */
     public function fixNonvoidTags($html)
```

Comment:
I don't want to interfere in the return type in this method. This meths is not tested and widely used and it may cause some issues on depending code.